build.gradle

```diff
@@ -247,6 +247,14 @@ configure(opentelemetryProjects) {
         links 'https://docs.oracle.com/javase/8/docs/api/'
     }
 
+    if (JavaVersion.current().isJava8Compatible()) {
+        allprojects {
```

Comment:
You have an allprojects nested in the "loop" that already walks over all relevant projects. This is probably not what you want.

Reply:
You want to specify a project? like api?

Reply:
No, that's not what I mean. What I mean is that we are already in a code block that gets executed for all projects. If you nest allprojects inside, you will get your code executed like

* outer code (api)
  * code in allprojects(api)
  * code in allprojects(sdk)
  * code in allprojects(propagation)
* outer code (sdk)
  * code in allprojects(api)
  * code in allprojects(sdk)
  * code in allprojects(propagation)

See also my other comment https://github.com/open-telemetry/opentelemetry-java/pull/1494#discussion_r465105481

Reply:
See here: https://github.com/dengliming/opentelemetry-java/blob/v0.7.0/build.gradle#L19

Reply:
👍 thanks.